src/tasks/processBulkUpload.ts

```diff
@@ -0,0 +1,18 @@
+import { getLogger } from '../logger';
+import { isProcessBulkUploadJobPayload } from '../types';
+import { InternalValidationError } from '../errors';
+
+const logger = getLogger(__filename);
+
+export const processBulkUpload = async (
+  payload: unknown,
+): Promise<void> => {
+  if (!isProcessBulkUploadJobPayload(payload)) {
+    logger.debug(isProcessBulkUploadJobPayload.errors, 'Malformed bulk upload job payload');
+    throw new InternalValidationError(
+      'The bulk upload job payload is not properly formed',
+      isProcessBulkUploadJobPayload.errors ?? [],
+    );
+  }
+  logger.debug(`Started processBulkUpload Job for Bulk Upload ID ${payload.bulkUploadId}`);
```

Comment:
Likewise, the [logging documentation](https://worker.graphile.org/docs/library/logger) recommends using `helpers.logger` rather than using the main logger directly, so that it can add context like the worker, task, and job ID. (The `helpers` object is the second [argument to tasks](https://worker.graphile.org/docs/tasks#task-executor-function).)